packages/api/src/hooks/useActiveLinkedToTradingAccount.ts

```diff
@@ -7,9 +7,11 @@ const useActiveLinkedToTradingAccount = () => {
     const { data: account_list_data } = useAccountsList();
     const { data: wallet_account_data } = useActiveWalletAccount();
 
-    const linkedLoginid = wallet_account_data?.linked_to?.find(linked => linked.loginid)?.loginid;
+    const linkedDtradeLoginId = wallet_account_data?.linked_to?.find(
```

Comment:
@thisyahlen-deriv shouldn't it be something like `linkedDerivAppsLoginId` instead?

Reply:
And why specifically looking for `linked?.platform === 'dtrade'`?

Reply:
Because thats what we wanna show in the deriv apps section and it must always be dtrade.

Reply:
It can be multiple linked login ids like from dxtrade, mt5 and ctrader. Thats why im targeting dtrade 😸

Reply:
Got it. `dtrade` is the name for trading account which is the Deriv apps section in a wallet.